### PR TITLE
2:  Runbook nudges

### PR DESCRIPTION
**Pull request by acsauk**
It's taken a ridiculous amount of time but here's the runbook nudge ticket ready for review!

I had to add a step for loading env vars in `base.py` - it looks like the command does nothing when the file being loaded doesn't exist so we should be good in prod. 

I've added a couple of super simple unit tests for the keyword handler code here more as a starting point on getting the testing process started. Once I'm a bit more familiar with python and Django we can look at adding the required pipeline stages but its a starting point for now.

As noted, I have 0 experience with this language and framework so give me all the feedback please 🙏

### CODE REVIEW

File: opgincidentresponse/actions/keyword_handlers.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-import logging
-logger = logging.getLogger(__name__)
-
 @keyword_handler(['status page', 'statuspage'])
-def status_page_notification(comms_channel: CommsChannel, user: str, keyword: str, text: str, ts: str):
+def status_page_notification(comms_channel: CommsChannel):
     comms_channel.post_in_channel(f"ℹ️ You mentioned the Status Page - You can find our statuspage here: https://theofficeofthepublicguardian.statuspage.io/")
+
+@keyword_handler(['status page', 'statuspage'])
```
Comment (gregtyler) [inline opgincidentresponse/actions/keyword_handlers.py:10]: Presumably this should hang off runbook keywords?
```suggestion
@keyword_handler(['run book', 'runbook'])
```

File: opgincidentresponse/settings/base.py [hunk 2]
```diff
@@ -4,6 +4,12 @@
 from django.core.exceptions import ImproperlyConfigured
 
 from response.slack.client import SlackClient
+from pathlib import Path
+from dotenv import load_dotenv
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+env_path = current_path / "../../.env"
+load_dotenv(dotenv_path=env_path)
```
Comment (gregtyler) [inline opgincidentresponse/settings/base.py:12]: This shouldn't be necessary if you have `.env` in the root directory

Comment (acsauk) [reply opgincidentresponse/settings/base.py:12]: Seeeee I was getting all sorts of issues running tests via docker so I reverted to using a virtualenv locally (.env at root in both instances).

File: opgincidentresponse/actions/keyword_handlers.py (hunk 1 above)
Comment (acsauk) [reply opgincidentresponse/actions/keyword_handlers.py]: Yeah realised once I was testing this locally just now 🤦  updated!

File: opgincidentresponse/settings/base.py (hunk 2 above)
Comment (gregtyler) [reply opgincidentresponse/settings/base.py:12]: Ah, then you probably need to do something like `source .env` to get the environment variables into memory.

The `.env` is just a convenient way to store then, not something that should be loaded into the application.

Comment (gregtyler) [reply opgincidentresponse/settings/base.py:12]: Ah, then you probably need to do something like `source .env` to get the environment variables into memory.

The `.env` is just a convenient way to store then, not something that should be loaded into the application.

**Review by gregtyler — APPROVED** [review]

**Pull request merged**